### DISCUSSION TITLE
Installtion fails caused by non existend files

### DISCUSSION THREAD

**Discussion by crackscout123** [JulianPrieber/littlelink-custom#377] [Issues]
Platform: `Ubuntu 18 LTS`
Server type: `Apache2`
PHP version: `8.2`

The installer is looking for two files which dont exsist called "INSTALLING" and "INSTALLERLOCK". 
I tried to work around it by creating a empty file called "INSTALLERLOCK" this works in the first place, I can go to the next step BUT now its looking for the "INSTALLING" file. I tried the same method with the empty file (and added permission so it can be accessed) but it does not work.

Here the Exception:
https://flareapp.io/share/W7zRk3k7#F55

**Comment by JulianPrieber** [comment]
Your file permissions aren't set correctly.
Make sure all files are owned by the Apache group and user. In Ubuntu in most cases this is the www-data group and user.

`sudo chown -R www-data:www-data /path/to/littlelink-custom`

Next enabled all members of the www-data group to read and write files

`sudo chmod -R g+rw /path/to/littlelink-custom`
